bindings/angular2/examples/alert-dialog.ts

```diff
@@ -3,73 +3,79 @@ import {
   Component,
   ComponentRef,
   AlertDialogFactory,
-  OnsAlertDialog,
   ViewChild,
-  AfterViewInit
+  ONS_DIRECTIVES,
+  Params,
+  OnInit,
+  OnDestroy
 } from '../src/angular2-onsenui';
 
 @Component({
   selector: 'div',
-  directives: [OnsAlertDialog],
+  directives: [ONS_DIRECTIVES],
```

Comment:
Yes, it should be NgModule. I will update NgModule related code on other pull-request.